Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
-      # set log level temporarily for npm's `make install`
-      ENV["NPM_CONFIG_LOGLEVEL"] = "verbose"
-      # unset prefix temporarily for npm's `make install`
-      ENV.delete "NPM_CONFIG_PREFIX"
-
-      cd buildpath/"npm_install" do
-        system "./configure", "--prefix=#{libexec}/npm"
```

Comment:
Refs: https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b

* Using locally-installed tools without npm run-script: Could be used to refactor Formulas executing `node_modules/.bin/buildtool` to use `npx buildtool` instead
* everything else should be not usefull for formula authors